types/validator_set.go

```diff
@@ -731,9 +731,9 @@ func (vals *ValidatorSet) VerifyCommit(chainID string, blockID BlockID,
 // VerifyCommitLight verifies +2/3 of the set had signed the given commit.
 // It does NOT count all signatures.
 func (vals *ValidatorSet) VerifyCommitLight(chainID string, blockID BlockID,
```

Comment:
Maybe it makes sense to add a new method `VerifyCommitLightWithCache` to avoid breaking the API